Assets/_BForBoss/_Utility/Scripts/Extensions.cs

```diff
@@ -0,0 +1,21 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Perigon.Utility
+{
+    public static class Extensions
+    {
+        public static T GetRandomElement<T>(this IList<T> array)
+        {
+            return array[Random.Range(0, array.Count)];
```

Comment:
We actually have a utility withing RanomUtilitiy that get's a random element in an array

Reply:
Oh cool thanks, I checked IListExtensions but forgot about that one!